tests/gtests/sw_maintest.cc

```diff
@@ -31,6 +31,7 @@
    of the SOILWAT2 repository
 */
 const char * dir_test = "./tests/example";
+const char * masterfile_test = "files.in"; // relative to 'dir_test'

```

Comment:
We have now `const char * masterfile_test = "files.in";` in "sw_maintest.cc" and `const char * filemaster_test = "files.in";` in "sw_testhelpers.cc" -- could we get rid of one? E.g., move it to "sw_testhelpers.h" and use in both places? And rename to something like `const char * filemain_test = "files.in";` when we are at it?